.github/workflows/link-check-deploy.yml

```diff
@@ -2,47 +2,58 @@ name: Check new links against deployment
 # This workflow "triggers" and skips on deployment because GitHub Actions /
 # Checks refuses to show the check on deployment_status
 on:
-  - deployment
   - deployment_status
 jobs:
   run:
-    name: Initialize
+    name: Run
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
Lets us continue even if an existing report comment isn't found

Reply:
Could this also skip any other error occurrence and skip without reporting the error?

Reply:
Could this also skip any other error occurrence without reporting the error?

Reply:
It seems like it does, I'll need to improve that next. If an error occurs, the script will just write the "Link Check Report" `h1` with no content. I'm pretty sure that can be fixed by getting stderr output into the content body in the bash script part of the action.